### DISCUSSION TITLE
[Feature] Delete Media Request for partner sharing

### DISCUSSION THREAD

**Discussion by TriMonDev** [immich-app/immich#12881] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, been loving immich so far!

Just wanted to ask for a feature that could be implemented as a request system within the "utilities" section of the web interface or wherever makes sense.
This request system could be used to manage what media should be deleted or not by requests of other users within the server, in my case i've mass uploaded some old photo backups for a family member and i've been seeing them in my memories tab often and would like to ask them to delete the media; but each time i do, i forget which photos to actually delete and asking the user with the photos uploaded to their share of the server is complicated as they are not an avid tech person, and searching through the large collection is not always consistent.

So, i would like to be able to send them a delete request or an archive request (if they want to keep the media but not share it) when i am on the photo/vid, or even mass select the media in question just like other options when mass selection a portion of the library.

Thank you for reading and i hope my request could be considered.
Have a great day/night!

### Platform

- [x] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can just send them the link to the full size view of the asset, building a whole request system for this seems unnecessarily complicated to me.

**Reply by TriMonDev** [reply]
Fair enough, thanks for the consideration.
